src/main/java/com/redislabs/redistimeseries/RedisTimeSeries.java

```diff
@@ -404,7 +405,46 @@ public long add(
       boolean uncompressed,
       Map<String, String> labels) {
     try (Jedis conn = getConnection()) {
-      byte[][] args = tsAddArgs(sourceKey, timestamp, value, retentionTime, uncompressed, labels);
+      byte[][] args =
+          tsAddArgs(sourceKey, timestamp, value, retentionTime, uncompressed, null, null, labels);
+      return sendCommand(conn, Command.ADD, args).getIntegerReply();
+    }
+  }
+
+  /**
+   * TS.ADD key timestamp value [RETENTION retentionTime] UNCOMPRESSED [CHUNK_SIZE size]
+   * [ON_DUPLICATE policy] [LABELS label value..]
+   *
+   * @param sourceKey
+   * @param timestamp
+   * @param value
+   * @param retentionTime
+   * @param chunkSize
+   * @param duplicatePolicy
+   * @param uncompressed
+   * @param labels
+   * @return
+   */
+  public long add(
+      String sourceKey,
+      long timestamp,
+      double value,
+      long retentionTime,
+      boolean uncompressed,
+      long chunkSize,
+      DuplicatePolicy duplicatePolicy,
+      Map<String, String> labels) {
+    try (Jedis conn = getConnection()) {
+      byte[][] args =
+          tsAddArgs(
+              sourceKey,
+              timestamp,
+              value,
+              retentionTime,
+              uncompressed,
+              chunkSize,
+              duplicatePolicy,
+              labels);
```

Comment:
@gkorland @filipecosta90 Would you please re-evaluate google formatter? This is making the code way too long.

Reply:
Let's discuss this on a specific PR. Any format will do ( as long we follow it and enforce it on CI ). Please submit an alternative that you find most suited.